draft-specifications/Model.md

```diff
@@ -179,6 +218,10 @@ Selectors cannot contain lone/unpaired surrogates.
 >   ```
 > - **Python**: `Selector('name')` where `Selector` is imported from `ocapn`.
 >
+> A selector's content must be expressible in UTF-8.
+> Strings in UTF-16 can express unpaired/lone surrogates that cannot be
+> expressed in any UTF and so cannot be carried by OCapN.
+>
 > Tracking: https://github.com/ocapn/ocapn/issues/46
 >
 > OCapN uses the name selector to avoid the implication that they will
```

Comment:
```suggestion
> OCapN uses the name `selector` to avoid the implication that they will
```

Reply:
Chose to TitleCase as that’s the prevailing style.

Reply:
What?

Reply:
I removed the back-ticks and promoted the case of “selector” to “Selector” to denote a pronoun, which I’m now using consistently to distinguish casual nouns from nouns with OCapN definitions.